rox bugfix for ptheme re #2277

01micko opened the pull request:
if rox is running and theme is changed rox is restarted.

Commit:
rox bugfix for ptheme re #2277

Commit:
add directrory-remote.svg to Puppy Standard and Puppy Flat icon themes

re #2277

Commit:
add diricons.sh to ptheme for fancy home directories in Rox

dimkr commented:
How did it work in old Puppies in the 5.x and 6.x era? 🤔

01micko commented:
> How did it work in old Puppies in the 5.x and 6.x era?

It didn't.

dimkr commented on woof-code/rootfs-packages/ptheme/usr/sbin/diricons.sh:
```diff
@@ -0,0 +1,30 @@
+
+for icon in $(ls "/usr/share/icons/$ICON_THEME/48/places/" | grep "^directory");do
+	if echo $icon|grep -q 'directory-home' ;then
+			ln_diricon "$HOME" "/usr/share/icons/$ICON_THEME/48/places/$icon"
```
Is it possible to link to the .svg and let ROX-Filer resize to 48 when it renders the icon?

01micko replied:
All icons _are_ svg in the native puppy themes.

Commit:
Merge branch 'testing' into ptheme

Commit:
Merge branch 'testing' into ptheme

Commit:
Merge branch 'testing' into ptheme

Commit:
Bugfixes for ptheme

- filter icon themes differently as DMZ (cursor) was showing up
- improved diricon script and renamed - now supports more icon themes
- improved gui in ptheme_gtk (gtk3 missing icons)
- minor tweaks
